tools/tidb-binlog-cluster.md

```diff
@@ -0,0 +1,566 @@
+---
+title: TiDB-Binlog cluster 版本用户文档
+category: advanced
+---
+
+> cluster 版 TiDB-Binlog 尚未发布正式版，本文档用于测试环境部署。
+
+# TiDB-Binlog 简介
+
+TiDB-Binlog 是一个用于收集 TiDB 的 Binlog，并提供实时备份和同步功能的商业工具。
+
+TiDB-Binlog 支持以下功能场景:
+
+* **数据同步**：同步 TiDB 集群数据到其他数据库
+* **实时备份和恢复**：备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复
+
+## TiDB-Binlog 架构
+
+首先介绍 TiDB-Binlog 的整体架构。
+
+![TiDB-Binlog 架构](../media/tidb_binlog_cluster_architecture.png)
+
+TiDB-Binlog 集群主要分为 Pump 和 Drainer 两个组件：
+
+### Pump
+
+Pump 用于实时记录 TiDB 产生的 Binlog，并将 Binlog 按照事务的提交时间进行排序，再提供给 Drainer 进行消费。
+
+### Drainer
+
+Drainer 从各个 Pump 中收集 Binlog 进行归并，再将 Binlog 转化成 SQL 或者指定格式的数据，最终同步到下游。
+
+## 主要特性
+* 多个 Pump 形成一个集群，可以水平扩容；
+* TiDB 通过内置的 pump client 将 Binlog 分发到各个 Pump；
+* Pump 负责存储 Binlog，并将 Binlog 按顺序提供给 Drainer；
+* Drainer 负责读取各个 Pump 的 Binlog，归并排序后发送到下游。
+
+
+# TiDB-Binlog 部署
```

Comment:
I think we should use the second-level heading here. Otherwise, please use "Pump."